GPG_KEY_SETUP_GUIDE.md

```
@@ -0,0 +1,226 @@
+- **GPG_PASSPHRASE**: The passphrase you set
+- **GPG_KEY_ID**: The 40-character fingerprint (or last 16 characters)
+
+---
+
+## Step 4: Add Secrets to GitHub Repository
+
+You have two options to add secrets:
+
+### Option A: Using GitHub Web Interface (Recommended)
+
+1. Go to: https://github.com/nodejs/nodejs-unofficial-builds/settings/secrets/actions
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)